ledger-tool/src/main.rs

```diff
@@ -82,6 +86,29 @@ fn main() {
         .subcommand(SubCommand::with_name("print").about("Print the ledger"))
         .subcommand(SubCommand::with_name("json").about("Print the ledger in JSON format"))
         .subcommand(SubCommand::with_name("verify").about("Verify the ledger's PoH"))
+        .subcommand(SubCommand::with_name("prune").about("Prune the ledger at the block height").arg(
+            Arg::with_name("valid_slot_list")
```

Comment:
nit: just remove "valid" from slot_list/slot-list and the `.help()`.  I feel "valid" is implicit, why give an "invalid" list :)